### PR TITLE
Fixes #35283 - populate templates description from metadata

### PR DESCRIPTION
**Pull request by nofaralfasi**
This PR applies a simpler solution for the missing description issue(compared to this [PR](https://github.com/theforeman/foreman_discovery/pull/578)), and it is intended for Satellite 6.12.

### CODE REVIEW

File: db/seeds.d/50_discovery_templates.rb [hunk 1]
```diff
@@ -22,6 +22,10 @@
...
     tmpl.organizations = organizations
     tmpl.locations = locations
+
+    metadata = Template.parse_metadata(content)
+    tmpl.description = metadata['description']
+
     tmpl.save!(:validate => false) if tmpl.changes.present?
   end
-end
+end
```
Comment (ares) [inline db/seeds.d/50_discovery_templates.rb:31]: ideally configure your editor so ti does not modify the EOL

**Comment by ares** [comment]
[test foreman_discovery] the failures seems unrelated but other PR seems to have them green

**Review by ares — COMMENTED** [review]
a minor thing, otherwise the fix makes sense

**Review by stejskalleos — APPROVED** [review]
Failed tests are not related, probably due to Foreman version

**Comment by stejskalleos** [comment]
**LGTM**, thanks @nofaralfasi !

**Pull request merged**